__tests__/api_client.test.ts

```diff
@@ -11,6 +11,7 @@ describe('ApiClient', () => {
     credentialsToken: 'yyy',
     dependabotApiUrl: 'https://localhost',
     dependabotApiDockerUrl: 'https://localhost',
+    updaterImage: '', // irrelevant for this test
```

Comment:
Will it still work if this input isn't provided at all? We'll want to make sure it stays compatible with older GHES versions which won't be populating this.

Reply:
Yes, that's done in this or statement: https://github.com/github/dependabot-action/pull/401/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR47-R48

If `params.updaterImage` is falsy (undefined, null, or empty string) then it falls back to the pinned version.

Reply:
That link doesn't work unless opened in a new tab, weird! It's this line in main.ts:

```typescript
    // The dynamic workflow can specify which updater image to use. If it doesn't, fall back to the pinned version.
    const updaterImage = params.updaterImage || UPDATER_IMAGE_NAME
```

Reply:
@mctofu I added explicit tests around which version of the updater is used.